packages/data-controls/src/test/index.js

```diff
@@ -31,9 +31,21 @@ describe( 'controls', () => {
 		const selectorWithFalseResolver = jest.fn();
 		selectorWithFalseResolver.hasResolver = false;
 		const hasFinishedResolution = jest.fn();
-		const unsubscribe = jest.fn();
-		let subscribedCallback;
+		const resolveSelectSelectorWithResolver = jest.fn();
+		const resolveSelectselectorWithUndefinedResolver = jest.fn();
+		const resolveSelectselectorWithFalseResolver = jest.fn();
+
 		const registryMock = {
+			__experimentalResolveSelect: ( storeKey ) => {
```

Comment:
This makes the test hard to follow.

`__experimentalResolveSelect` and `select` should share the same `mockStore` and as many mock functions as possible.

Reply:
Uodated 👍